server/services/interventionsService.test.ts

```diff
@@ -1552,17 +1552,17 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
     const actionPlanAppointments = [
       {
         sessionNumber: 1,
-        appointmentTime: '2021-05-13T12:30:00.000000Z',
+        appointmentTime: '2021-05-13T13:30:00+01:00',
```

Comment:
I'm a little confused about this change. What are we trying to communicate with this time zone? Does the time zone passed here have some meaning, and if so what is the frontend meant to pass?

Reply:
When implementing the contract tests server side a couple things came to light:

1) appointmentTime is in a period that is subject to BST (based on the local my server) and the serialisation library considers this when doing the serialisation, so whilst the time is materially the same, it serialises to a different string
2) if the milli's is set to zero, the component is omitted from the serialised form 

These changes were necessary to get the contract tests to function from the server side as it appears the verification is based on string values.

Reply:
> appointmentTime is in a period that is subject to BST (based on the local my server) and the serialisation library considers this when doing the serialisation, so whilst the time is materially the same, it serialises to a different string

So, in that case it makes sense that the API might return ISO 8601 timestamps with a non-zero UTC offset in the GET appointments response. But in the contract tests I'm also seeing examples of non-zero offsets being supplied to the POST and PATCH endpoints too. And I'd like to understand what you need this offset for, and if that means there's an expectation on the client to calculate this offset?

I'd say that _if_ you do need time zone information, then a clearer way of communicating it is a tuple like (date, wall time, tz identifier), e.g. `('2021-03-01', '10:03:00.000', 'Europe/London')`.